throttle/middleware_test_helper.go

```diff
@@ -71,3 +71,57 @@ func throttleMiddlewareHandler(ttm loomchain.TxMiddlewareFunc, state loomchain.S
 		false,
 	)
 }
+
+func throttleMiddlewareHandlerCheckTx(ttm loomchain.TxMiddlewareFunc, state loomchain.State, tx auth.SignedTx, ctx context.Context) (loomchain.TxHandlerResult, error) {
+	return ttm.ProcessTx(
+		state.WithContext(ctx),
+		tx.Inner,
+		func(state loomchain.State, txBytes []byte, isCheckTx bool) (res loomchain.TxHandlerResult, err error) {
+
+			var nonceTx loomAuth.NonceTx
```

Comment:
Why do we need to simulate this entire thing? We only care if tx reaches `next` or not